### ISSUE TITLE
Telemetry data is not being recorded and displayed after test completion with Multi-Server setup 

### ISSUE THREAD

**Issue by ttrading** [librespeed/speedtest#416]
## Description
We have a two server setup in Azure environment using Ubuntu 18.04. One of the servers host the front-end, back-end and PostgreSQL database. The second server hosts the backend only. Servers are geographically separated. We are using "example-multipleServers-full.html" as a template on the server where we have the front-end. We can connect to the front-end, select a server and run the test. We can see the results on the gauges and score cards. However, once the test is over, the system is not load the image with the results and there are no records in the PostgreSQL table. Please note that we created a dedicated psql account for LibreSpeed on the server that hosts PostgreSQL. This account is the owner of the "speed" database which is used to keep the telemetry data. PostgreSQL is configured to allow incoming TCP/IP connections.

## Server
Server version: Apache/2.4.29 (Ubuntu) || psql (12.6 (Ubuntu 12.6-1.pgdg18.04+1)) || PHP 7.2.24-0ubuntu0.18.04.7 (cli) (built: Oct  7 2020 15:24:25) ( NTS )

## Client
Chrome 89

## Steps to reproduce
* Do this
* Do that
* Bad stuff happens

## Expected behaviour
Installation instructions have been taken from: https://github.com/librespeed/speedtest/blob/master/doc.md
Basically, the server with the front end has a similar install to single server setup except we use the multi-server full.html ad use postgresql. The second server has the backend files under /var/www/html. Both systems are built based on ubuntu 18.04 from Azure.
We are expecting to see an image underneath the gauges showing the test results just like the single server setup & telemetry records in the dedicated psql table.

**Comment by adolfintel** [comment]
If results are not stored in the database at all, check apache's error.log file, you should see an error in telemetry.php

**Comment by ttrading** [comment]
I did the installation again with mysql and switched to postgresql later on. I must have done something wrong before because everything worked this time. The only issue I am noticing under the developer tools when I run a test is this message:

**Access to XMLHttpRequest at 'http://XXXXXXX1.cloudapp.azure.com/backend/empty.php' from origin 'http://XXXXXXX2.cloudapp.azure.com/backend/empty.php' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.**

it is listed as an error. Maybe this should be handled as part of another ticket. Otherwise, everything has been resolved. Closing the ticket.
